predicators/spot_utils/skills/spot_sweep.py

```diff
@@ -44,9 +44,11 @@ def sweep(robot: Robot, sweep_start_pose: math_helpers.SE3Pose, move_dx: float,
     # Stow arm.
     stow_arm(robot)
     # Back up a little bit so that we can see the result of sweeping.
+    # NOTE: this is unfortunately world-frame specific because I'm too dumb to
+    # figure out how to do this properly.
     body_rel_pose = math_helpers.SE2Pose(
-        x=-0.2,
-        y=0.0,
+        x=-0.1,
+        y=-0.3,
```

Comment:
hold on `body_rel_pose` is in the robot's frame, so wouldn't this just generalize to any world frame?

Reply:
good point, i'll clarify the comment. the thing that is hard-coded here is the assumption that sweeping is towards the right side of the robot